modules/memberships/class-jetpack-memberships.php

```diff
@@ -63,6 +71,7 @@ public static function get_instance() {
 		if ( ! self::$instance ) {
 			self::$instance = new self();
 			self::$instance->register_init_hook();
+			self::$required_plan = ( defined( 'IS_WPCOM' ) && IS_WPCOM ) ? 'value_bundle' : 'jetpack_premium';
```

Comment:
I believe the lowest plan should be enough? I.e. personal, no need premium?

Reply:
Good point, oversight on my part.